src/user/views/user_views.py

```diff
@@ -72,6 +78,47 @@ def get_serializer_class(self):
     def get_serializer_context(self):
         return {"get_subscribed": True, "get_balance": True, "user": self.request.user}
 
+    @action(detail=False, methods=["GET"], permission_classes=[AllowAny])
```

Comment:
Here we get a list of invited users and the amount of rsc earned by referring them